buildkit/0-buildkitd.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: buildkitd
+  name: buildkitd
+spec:
+  selector:
+    matchLabels:
+      app: buildkitd
+  template:
+    metadata:
+      labels:
+        app: buildkitd
+    spec:
+      containers:
```

Comment:
Can one instance of this be used across multiple namespaces?

I'm thinking forward to the world where we have `ClusterBuildTemplate` in a knative/build release and can run a single `buildkitd` per cluster that are shared by all builds.  Does that make sense?